docs/primer/scopeandkeyconcepts.md

```diff
@@ -22,25 +22,21 @@ The primary focus of OFDS is to describe the passive network infrastructure. The
 
 ### Actors
 
-Based on the layers in the network value chain, there are three main actors in a fibre optic network:
+Based on the layers in the network value chain, there are three main groups of actors in a fibre optic network:
 
 ![The actors in fibre optic networks](../../_assets/actors.svg)
 
 #### Physical infrastructure provider
 
 OFDS defines a physical infrastructure provider as:
 
-```{jsoninclude-quote} ../../schema/network-schema.json
-:jsonpointer: /definitions/Link/properties/physicalInfrastructureProvider/description
-```
+>An organisation that owns and maintains passive network infrastructure, i.e. the non-electrical elements, such as dark fibre, ducts and physical sites.
```

Comment:
Hardcoding the code descriptions is fine for this PR but creates a SSOT issue going forward. I've opened an issue: https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/142

Reply:
yep good plan